internal/services/network/express_route_connection_resource.go

```diff
@@ -71,6 +71,12 @@ func resourceExpressRouteConnection() *pluginsdk.Resource {
 				Optional: true,
 			},
 
+			"enable_private_link_fast_path": {
```

Comment:
Please rename this to 
```suggestion
			"private_link_fast_path_enabled": {
```

Reply:
Renamed in all places.